EasyCommands.Tests/ScriptTests/FunctionalTests/SimpleComparisonsBooleanTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace EasyCommands.Tests.ScriptTests {
+    [TestClass]
+    public class SimpleComparisonsBooleanTests {
+        [TestMethod]
+        public void LessThan() {
+            var lines = new List<String>{ "False < True", "True < False", "False < False", "True < True" };
+
+            using (var test = new SimpleExpressionsTest(lines)) {
```

Comment:
I hate to do this to you, but I think SimpleExpressionsTest is unnecessary.  I'd prefer to follow the pattern used in all the other tests.

Instead, you can do the following with equal effect:

var script = @"
print false < true
print true < false
print false < false
print true < true
";

Reply:
the main point is to get the proper message for the fail messages in the asserts.

Reply:
When it comes to test, I actually don't mind that the script is exactly what you're testing against, vs having a layer of separation from the actual compiled test.  The extra stuff is just the "print" statement in this case, haha.

Reply:
Do you want it changed?

Reply:
I would like it changed, but I can handle as part of re-doing some the Operations tests.